.chef/knife.rb

```diff
@@ -0,0 +1,17 @@
+# See http://docs.opscode.com/config_rb_knife.html
+# for more information on knife configuration options
+
+current_dir = File.dirname(__FILE__)
+log_level :info
+log_location STDOUT
+node_name 'fakeclient'
+client_key "#{current_dir}/fakeclient.pem"
```

Comment:
Did you just `touch` these files locally? If so, you'll want to commit them as an empty file. You can do that with

```bash
git add -N .chef/fakeclient.pem
git add -P .chef/fakeclient.pem
```

And answer yes to staging the empty hunk.